src/h5web/visualizations/heatmap/HeatmapVis.module.css

```diff
@@ -30,6 +30,29 @@
   pointer-events: none;
 }
 
+.textureLoader {
+  background: rgba(0, 0, 0, 0.4)
+    linear-gradient(
+      to right,
+      transparent,
+      rgba(255, 255, 255, 0.5) 60%,
+      transparent
+    ) -50% top / 35% 100% no-repeat;
+  transition: opacity 0.2s ease-out;
+  opacity: 0;
+}
+
+.textureLoader[data-visible] {
+  opacity: 1;
+  animation: loading 1s ease-in-out infinite;
+}
+
+@keyframes loading {
+  to {
+    background-position-x: 150%;
```

Comment:
The gradient, which takes 35% of the width of the canvas, moves from -50% to 150% in a loop.